automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala

```diff
@@ -257,9 +257,10 @@ class NotebookClusterMonitoringSpec extends GPAllocFixtureSpec with ParallelTest
                                              welderDockerImage = Some(LeonardoConfig.Leonardo.oldWelderDockerImage))
       ) { cluster =>
         // Verify welder is running with old version
-        val statusResponse = Welder.getWelderStatus(cluster).unsafeRunSync()
+        val statusResponse = Welder.getWelderStatus(cluster).attempt.unsafeRunSync()
```

Comment:
Using `attempt` for better error reporting. Otherwise it throws a hard-to-understand `circe` exception.